game/src/Autoload/SoundManager.gd

```diff
@@ -0,0 +1,36 @@
+extends Node
+
+const _audio_directory_path : StringName = &"res://audio/sfx/"
+
+var _loaded_sound : Dictionary = {}
+
+var _bus_to_stream_player : Dictionary = {}
+
+func _ready():
+	var dir = DirAccess.open(_audio_directory_path)
+	for fname in dir.get_files():
+		match fname.get_extension():
+			"ogg", "wav":
```

Comment:
If we're going to support ogg and wav we might as well load mp3's too if only for the flexibility.